sources/datasources/users.js

```diff
@@ -83,10 +82,9 @@ class UserAPI extends DataSource {
 		retPromise = await updatedUser.save();
 		return retPromise;		
 	}
-
 	async deleteUser(uid) {
-		const foundUser = await Users.find({firebaseUID:uid});;
-		return await foundUser.deleteOne();
+		await Users.deleteOne({firebaseUID:uid});
+		return {success:true};
```

Comment:
Why is the newer code better over previous one?
In older one, I suppose the client will get the deleted user's response in API and in new one just success:true and in case of error, there will be a Mongoose error a timeout?

Reply:
We arent sure what response is expected after delete operation, so simply we're returning a success:true. 
Error Handling needs to be done extensively at a lot of places, thanks for pointing it out, we'll do that 🐼